### PR TITLE
Add IMS Toucan to model-libraries.ts

### PR DESCRIPTION
**Pull request by Flux9665**
We have moved the models needed to run our TTS toolkit to the hub. We would like to track the downloads, therefore this PR adds IMS Toucan and specifies countDownloads to track our .pt files.

### CODE REVIEW

File: packages/tasks/src/model-libraries.ts [hunk 1]
```diff
@@ -661,6 +661,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: true,
 		countDownloads: `path:"pytorch_model.bin" OR path:"model.safetensors"`,
 	},
+	imstoucan: {
```
Comment (Wauplin) [inline packages/tasks/src/model-libraries.ts:664]: There is currently no models tagged as `imstoucan` (see [here](https://huggingface.co/models?other=imstoucan)). Could you add `library_name: imstoucan` to the modelcard metadata of https://huggingface.co/Flux9665/ToucanTTS to reference it properly?

File: packages/tasks/src/model-libraries.ts [hunk 2]
```diff
@@ -661,6 +661,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
...
+		prettyLabel: IMS-Toucan",
+		repoName: "IMS-Toucan",
+		repoUrl: "https://github.com/DigitalPhonetics/IMS-Toucan",
+		countDownloads: `path_extension:"pt"`,
```
Comment (Wauplin) [inline packages/tasks/src/model-libraries.ts:668]: For what I can see, https://huggingface.co/Flux9665/ToucanTTS/tree/main has a lot of files including two `.pt` files ([16kHz_encodec.pt](https://huggingface.co/Flux9665/ToucanTTS/blob/main/16kHz_encodec.pt) and [embedding_gan.pt](https://huggingface.co/Flux9665/ToucanTTS/blob/main/embedding_gan.pt)). This means that if the full repo is downloaded, we would be counting twice the number of downloads. In such a case it's preferable to set `path: "..."` with only one specific file. So the question you should wonder is "which files is guaranteed to be downloaded when a user instantiates the model?".

**Review by Wauplin — COMMENTED** [review]
Thanks for opening this PR @Flux9665 ! I've added 2 comments to address before merging. Also, would it be possible to move `imstoucan` up in the file? We try to keep the libraries definitions alphabetically ordered in the mapping. Thanks in advance!

File: packages/tasks/src/model-libraries.ts (hunk 1 above)
Comment (Flux9665) [reply packages/tasks/src/model-libraries.ts:664]: Thanks for the hint, I added the library_name to the modelcard.

File: packages/tasks/src/model-libraries.ts (hunk 2 above)
Comment (Flux9665) [reply packages/tasks/src/model-libraries.ts:668]: The full repo is never downloaded, in the toolkit I only use hf_hub_download to download individual models from this bundle on demand. 

People might only want to use the pretrained vocoder, but train their own ToucanTTS checkpoint, or the other way around. That's why I chose the path_extension as filter. But you are right, it is too broad.

Is it ok to include more than one file? Then I would include the three most important files in the path. At least one of them will be downloaded by most users, but rarely more than one of them.

Comment (Wauplin) [reply packages/tasks/src/model-libraries.ts]: That's perfectly valid and seems the most reasonable thing to do in your case yes. Here is an example on how to do it: https://github.com/huggingface/huggingface.js/blob/18231b70bd5435444ef3e1903b83a911dca1b374/packages/tasks/src/model-libraries.ts#L290

Comment (Flux9665) [reply packages/tasks/src/model-libraries.ts]: Thank you! It is fixed now.

File: packages/tasks/src/model-libraries.ts [hunk 3]
```diff
@@ -301,6 +301,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/Tencent/HunyuanDiT",
 		countDownloads: `path:"pytorch_model_ema.pt" OR path:"pytorch_model_distill.pt"`,
 	},
+	imstoucan: {
+		prettyLabel: "IMS Toucan",
+		repoName: "IMS-Toucan",
+		repoUrl: "https://github.com/DigitalPhonetics/IMS-Toucan",
+		countDownloads: `path:"embedding_gan.pt" OR path:"Vocoder.pt", OR path:"ToucanTTS.pt"`,
```
Comment (Wauplin) [inline packages/tasks/src/model-libraries.ts:308]:
```suggestion
		countDownloads: `path:"embedding_gan.pt" OR path:"Vocoder.pt" OR path:"ToucanTTS.pt"`,
```

**Review by Wauplin — APPROVED** [review]
Looks good! Thanks @Flux9665 :hugs:

**Review by Vaibhavs10 — COMMENTED** [review]
Nice! Thanks @Flux9665 🤗

**Pull request merged**